templates/containers/lxd.html

```diff
@@ -77,9 +77,6 @@ <h2 class="p-muted-heading u-align--center">Supported guest operating systems</h
       <li class="p-inline-images__item">
```

Comment:
Does this page exist in the new IA? Shouldn't this maybe be deleted too? If not, just ignore me.